src/index.js

```diff
@@ -0,0 +1,182 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+// The top-level APIs offered by this library are `ClientConnection` and
+// `ServerConnection` classes.  They each take a callback to be used for
+// sending data to the remote peer, and operate like this:
+//
+//    conn = await ClientConnection.create(psk, pskID, async function send_data_to_server(data) {
```

Comment:
we can maybe use example syntax like https://github.com/mozilla/fxa-crypto-relier/blob/master/src/deriver/ScopedKeys.js#L19 , maybe

Reply:
`pskID` -> `pskId` ?